Sources/App/Migrations/080/UpdateBuildAddBuildErrors.swift

```diff
@@ -0,0 +1,31 @@
+// Copyright Dave Verwer, Sven A. Schmidt, and other contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Fluent
+
+
+#warning("move to proper folder before merging")
```

Comment:
This is in a folder `080` only so it doesn't conflict with the migration in branch `fix-2970`. Will move to proper sequence before merging, depending on which one ends up going first.